docfx_yaml/extension.py

```diff
@@ -486,7 +487,29 @@ def _parse_docstring_summary(summary):
 
         # Parse keywords if found.
         # lstrip is added to parse code blocks that are not formatted well.
-        if part.lstrip('\n').startswith('..'):
+        if (potential_keyword := part.lstrip('\n')) and (
+            potential_keyword.startswith('..') or
+            potential_keyword.startswith(enum_type_token)
```

Comment:
Can this just be included in the [`extract_keyword`](https://github.com/googleapis/sphinx-docfx-yaml/blob/e147b144129d449cab6de0a8ca13c2588a9c0730/docfx_yaml/extension.py#L370) logic?

Reply:
I thought about it, but unfortunately the entire section didn't get tokenized:
```
Values:
  name(val):
    ...
```
Other parts that do get tokenized and/or has proper format appear in this format:
```
.. note::
   ...

.. example::
   ...
```